src/redux/selectors/index.js

```diff
@@ -1,6 +1,34 @@
-export const getSearchDetails = state => state.imagesGalleryPage
+import { createSelector } from 'reselect'
+
+import { ITEMS_PER_PAGES } from 'common/constants'
+
+// General
+export const getIsLoading = state => state.imgDailyPage.isLoading
+
+//  Image gallery
+export const getImages = state => state.imagesGalleryPage.items
```

Comment:
1. Will be good te separate all simple selectors from `createSelector` selectors, I mean to have one file for this kind of selectors:
```
// selectors/index.js
export const getImages = state => state.imagesGalleryPage.items
```

and another file for that kind of selectors:
```
// selectors/common-selectors.js

import { getImages  } from './'

export const getImagesSelector = createSelector(
  getImages,
  items => items.flatMap(({ links }) => links),
)
```